.pre-commit-config.yaml

```diff
@@ -14,11 +14,11 @@ repos:
     hooks:
     -   id: flake8
 
--   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.931
-    hooks:
-        - id: mypy
-          additional_dependencies: [types-all]
+#-   repo: https://github.com/pre-commit/mirrors-mypy
+#    rev: v0.931
+#    hooks:
+#        - id: mypy
+#          additional_dependencies: [types-all]
```

Comment:
If we're not planning to use it, can it be completely removed instead of commented out?